ansible/playbooks/roles/repository/files/download-requirements/src/command/dnf.py

```diff
@@ -1,9 +1,13 @@
 from typing import Dict, List
+import re
 
 from src.command.command import Command
 from src.error import CriticalError
 
 
+def filter_non_critical_errors(stderr: str):
```

Comment:
I suggest creating class:
```python
class DnfBase(Command):
    def __init__(self, retries: int):
        super().__ini__('dnf', retries)

    def _filter_non_critical_errors(self, stderr: str):
        ....
```
Then:

```python
class Dnf(DnfBase):
```

So that each class could use this method inside without imports and also hidden behind interface.